oic.r.acl.raml

```diff
@@ -19,7 +19,25 @@ traits:
 
   get:
     description: |
-      Retrieves the acl data.
+      Retrieves the ACL data.
+      When used without query parameters, all the ACE entries are returned.
+      When used with a subjectuuid or resources query parameter, only the
+      ACEs matching the specified parameter are returned.
+    queryParameters:
+        subjectuuid:
+            type: string
+            description: |
+              Returns ACEs with the specificed subject UUID
+            required: false
+            repeat: false
+            example: se61c3e6b-9c54-4b81-8ce5-f9039c1d04d9
+        resources:
```

Comment:
Looking closer at the IoTivity code, "resources" is only processed in conjunction with subjectuuid. I'll try to get this expressed in the RAML.